java-gradle/README.md

```diff
@@ -20,7 +20,7 @@ This sample may be out of date. If you face any issues, please reach out to mail
      ```
 3. Run using gradle:
     ```bash
-    $ ./gradlew run
+    $ gradle run
```

Comment:
@ManuelNemirovsky I think @MickaelAlliel wanted it the other way around

Reply:
Done

Reply:
Working now.
`./gradlew fatJar run` needs to be run